usagov/vote-gov-drupal
# Vote-1152 -> User create and Delete work flow for testing

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1152](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1152&assignee=611272e0eb7aef00694b8a97) and [Vote-1153](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1153&assignee=611272e0eb7aef00694b8a97)
## Description (optional)
This Pr will create a new testing flow when it comes to functional testing.  The new flow will run the create users commands, run functional testing, and then delete after.  

I have also moved all tests into the testing folder and the `backend` tests are now just log in.  The script that was updated will run user scripts headless and both the create and delete user are now in one test file 

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. run `lando rebuild -y`
3. make sure your local is started 
4. cd into `testing`
5. run `npm run cy:test`
6. verify that the `create user` script runs, then function tests pass, then the `delete user` script runs. 
7. run `npm run cy:createUser` 
8. visit `http://vote-gov.lndo.site/user/login`
9. refer to the `cypress.config.js` lines 21-36 and try and log into each role -> expected result is you should be able to log in as all thee. 
10. return back to command line and run `npm run cy:deleteUser` 
11. repeat steps 8 and 9 -> expected result now should be no longer able to log in as user
